src/types/plugin-metadata.ts

```diff
@@ -3,6 +3,7 @@ import {KalturaPlayer} from '@playkit-js/kaltura-player-js';
 export interface PluginMetaData {
   pluginName: string;
   action: (player: KalturaPlayer) => void;
+  isEntrySupported: (player: KalturaPlayer) => boolean;
```

Comment:
duplicate of `isDisabled`
https://github.com/kaltura/playkit-js-audio-player/blob/master/src/types/plugin-metadata.ts#L10